01week/resume/css/style.css

```diff
@@ -1,3 +1,70 @@
 h1 {
-  font-family: 'Helvetica', sans-serif;
```

Comment:
I changed the font because I couldn't get it to work. It's because I didn't need the single quotes here.